packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -52,79 +52,87 @@
 			background: none;
 		}
 	}
-}
-
-.wp-block-navigation-menu .block-editor-block-list__layout,
-.wp-block-navigation-menu {
-	display: flex;
-	white-space: nowrap;
-}
 
-.wp-block-navigation-menu__inserter-content {
-	padding: $grid-size-large;
-}
+	.wp-block-navigation-menu .block-editor-block-list__layout,
+	.wp-block-navigation-menu {
+		display: flex;
+		white-space: nowrap;
+	}
 
-.wp-block-navigation-menu-item {
-	margin-left: $grid-size;
-	.wp-block-navigation-menu-item__link {
-		margin-left: $grid-size-large;
+	.wp-block-navigation-menu__inserter-content {
+		padding: $grid-size-large;
 	}
-}
 
-/**
- * Colors Selector component
- */
-$colors-selector-size: 22px;
-.block-library-colors-selector {
-	width: auto;
-
-	// Toolbar colors-selector button.
-	.block-library-colors-selector__toggle {
-		display: block;
-		margin: 0 auto;
-		padding: 3px;
-		width: auto;
+	// Handling item colors depending on the navigation.
+	.wp-block-navigation-menu {
+		&.has-background-color {
+			.wp-block-navigation-menu-item {
+				background-color: inherit; // it inherits the color defined in the navigation.
+			}
+		}
+
+		&.has-text-color {
+			.wp-block-navigation-menu-item {
```

Comment:
Same as above

Reply:
[Same as above ](https://github.com/WordPress/gutenberg/pull/18380#discussion_r344630452)for me as well.